Index: Turnin-NG/setup.py
===================================================================
--- Turnin-NG.orig/setup.py	2009-09-01 17:11:24.048828229 -0400
+++ Turnin-NG/setup.py	2009-09-01 17:53:16.268829341 -0400
@@ -190,6 +190,5 @@
       package_dir={'turninng':'src/turninng'},
       data_files=data_files,
       cmdclass={'build_infopage': build_infopage, 'build_pdf':build_pdf,
-          'build_htmldocs': build_htmldocs, 'build_legacy': build_legacy,
-          'install_legacy': install_legacy}
+          'build_htmldocs': build_htmldocs)
 )
